tests/experiment_params.yaml

```diff
@@ -1,18 +1,18 @@
-input_path: tests/data
+input_path: data
```

Comment:
This should be put back to `tests/data` for the tests to run properly

Reply:
I will change this